fmriprep/workflows/bold/util.py

```diff
@@ -412,3 +429,34 @@ def init_skullstrip_bold_wf(name='skullstrip_bold_wf'):
     ])
 
     return workflow
+
+
+def _pass_dummy_scans(algo_dummy_scans, dummy_scans):
+    """
+    **Parameters**
+
+    algo_dummy_scans : int
+        number of volumes to skip determined by an algorithm
+    dummy_scans : int or None
+        number of volumes to skip determined by the user
+
+    **Returns**
+    dummy_scans : int or None
+        number of volumes user specified to skip
+    algo_dummy_scans : int
+        number of volumes algorithm detected to skip
+    skip_vols_num : int
+        number of volumes to skip
+    """
+    from nipype import logging
+    LOGGER = logging.getLogger('nipype.interface')
+    if dummy_scans:
+        if algo_dummy_scans > dummy_scans:
+            LOGGER.warning("The non steady state algorithm detected "
+                           "more volumes than what was manually specified")
+
+        skip_vols_num = dummy_scans
+    else:
+        skip_vols_num = algo_dummy_scans
+
+    return dummy_scans, algo_dummy_scans, skip_vols_num
```

Comment:
I think returning unmodified values can be prone to error.
```suggestion
    return skip_vols_num
```

Reply:
good to know, my next set of commits should work around this.